### ISSUE TITLE
Widget lags behind around 4 months

### ISSUE THREAD

**Issue by petrarca-arezzo** [SimpleMobileTools/Simple-Calendar#1537]
The widget's date is around four months behind the current date (see screenshot). I have a brand new Android 8.1 and a new Simple-Calendar installation (version 6.15.4):
![Screenshot Calendar Widget](https://user-images.githubusercontent.com/74855438/140953274-e3d7faf3-9eb7-4a27-876d-c4d2b630eba9.png)

Side note - I have a widget of the Etar-Calendar as interim solution as well and this one shows the correct date.

**Comment by tibbi** [comment]
that is not the date, it just shows old events. You can change what past events are shown there in the app settings.

**Comment by petrarca-arezzo** [comment]
Yeah - I found it. But it's quite confusing as the settings is under calendar list-view and not under widget-configuration (where I looked first and where I expected such kind of setting).

**Comment by tibbi** [comment]
because it applies at the in-app list view too, not just the widget
